### PR TITLE
Try to load prisma.schema path from pkg json

### PR DESCRIPTION
**Pull request by Zeko369**
Closes: #3860

### What are the changes and their implications?

It first tries to load package.json prisma.schema value (i.e. if you have a `prisma/schema.prisma` folder instead of `db/schema.prisma`) instead of just using `db/schema.prisma`

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

File: packages/generator/src/generators/model-generator.ts [hunk 1]
```diff
@@ -41,7 +41,10 @@ export class ModelGenerator extends Generator<ModelGeneratorOptions> {
 
   // eslint-disable-next-line require-await
   async write() {
-    const schemaPath = path.resolve("db/schema.prisma")
+    const pkgJson = this.fs.readJSON("package.json", {}) as {[key: string]: any}
```
Comment (beerose) [inline packages/generator/src/generators/model-generator.ts:44]: Let's use `cwd`:
```
const packageJsonPath = path.join(process.cwd(), "package.json")
const packageJson = fs.readJSONSync(packageJsonPath)
```

File: packages/generator/src/generators/model-generator.ts [hunk 2]
```diff
@@ -41,7 +41,10 @@ export class ModelGenerator extends Generator<ModelGeneratorOptions> {
...
+    const rawSchemaPath = pkgJson?.["prisma"]?.["schema"] || "db/schema.prisma"
```
Comment (beerose) [inline packages/generator/src/generators/model-generator.ts:45]: To simplify:
```suggestion
    const rawSchemaPath = pkgJson?.prisma?.schema || "db/schema.prisma"
```

File: packages/generator/src/generators/model-generator.ts (hunk 1 above)
Comment (Zeko369) [reply packages/generator/src/generators/model-generator.ts:44]: The reason for using `this.fs.readJSON` was it loads the root path for us
(so i.e. running this inside of `pages` or `app` will work with `this.fs.readJSON`, but won't with `process.cwd()`)

Comment (beerose) [reply packages/generator/src/generators/model-generator.ts:44]: I see. it's fine then

**Comment by beerose** [comment]
Could you add a changeset too?

**Review by dillonraphael — APPROVED** [review]

**Pull request merged**